html_css/v2/main-course/custom-properties.md

```diff
@@ -0,0 +1,205 @@
+### Introduction
+
+Custom properties (also known as CSS variables) can be a really useful and powerful tool when writing our CSS files. In short, they allows us to reference a CSS value however many times we want throughout a file. How is this any different or better than just using the value, you ask?
```

Comment:
I'd leave out the rhetorical question here ("how is this different..") and just be direct 😄